assignments/arrays.js

```diff
@@ -63,34 +64,55 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`Car 33 is a ` +inventory[32].car_year + ' '+inventory[32].car_make+ ' ' +inventory[32].car_model );
+// console.log(inventory[32].car_year)
 
 
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
 let lastCar = 0;
-console.log();
+lastCar = inventory.pop();
+console.log(lastCar.car_make + ' '+lastCar.car_model);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+for (let i=0; i<inventory.length; i++) {
+    carModels.push(inventory[i].car_model);
+    (carModels.sort());
```

Comment:
You don't need the parens around methods that you chain on with dot notation. Its gonna end up muddling up your code and making it harder to read. There are times that you will have a hard enough time finding where the missing parenthesis is, without adding unnecessary ones. Trust me on this.